File: Algorithms/python/Two_Sum.py
def twoSum(nums, target):
    print(nums)
    for i in range(len(nums)):
        for  j in range(i, len(nums)):
            if nums[i] + nums[j] == target and i != j:
                return [i, j]
            elif i == len(nums) and j == len(nums):
                return []


print(twoSum([3,2,4], 6))

File: Algorithms/C++/TwoSum/twoSum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        map<int , int> mp;
        vector<int> v;
        for(int i=0;i<nums.size();i++){
            mp[nums[i]]=i;
        }

        for(int i=0;i<nums.size();i++){
            if(mp[(target)-nums[i]]&&mp[(target)-nums[i]]!=i){
                v.push_back(i);
                v.push_back(mp[target-nums[i]]);
                break;
            }
        }

        return v;
    }
};